src/main/flight/pid.c

```diff
@@ -1367,6 +1372,12 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
         }
 #endif // USE_YAW_SPIN_RECOVERY
 
+#ifdef USE_MAVLINK_ATTRATE
```

Comment:
This should use the Betaflight RC protocol stack and not require custom code in `pid.c`.